.ci/renewable-deploy/deploy-to-google-cloud.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -ex
+
+gcloud config set project ${GOOGLE_CLOUD_PROJECT_ID}
+gcloud config set compute/zone ${GOOGLE_CLOUD_ZONE}
+gcloud auth activate-service-account --key-file /tmp/infrastructure/google-cloud/service-account.json
+
+# when cluster is not up and running the result of `gcloud container clusters list` is empty
+IS_CLUSTER_UP=$(gcloud container clusters list --filter name=${GOOGLE_CLOUD_CLUSTER_NAME})
+if [ -z ${IS_CLUSTER_UP} ]; then
+  echo "Cluster \"${GOOGLE_CLOUD_CLUSTER_NAME}\" does not exist."
+  echo "Creating cluster..."
+  gcloud container clusters create ${GOOGLE_CLOUD_CLUSTER_NAME}
+  gcloud container clusters get-credentials ${GOOGLE_CLOUD_CLUSTER_NAME}
+  kubectl create namespace production
+  kubectl config set-context --current --namespace=production
+  kubectl create secret tls domain-ssl-certificates --cert=${TLS_CERT_PATH} --key=${TLS_KEY_PATH}
```

Comment:
I give it a second thought and what about to create following `tls-secret.yml` file
```
apiVersion: v1
kind: Secret
metadata:
  name: mg-secret-test
  namespace: production
data:
  tls.crt: {{TLS_CERT}}
  tls.key: {{TLS_KEY}}
type: kubernetes.io/tls
```

and before deploy replace pleaceholders with base64_encoded values like
```
sed "s/{{TLS_CERT}}/`cat $TLS_CERT_PATH|base64`/g" tls-secret.yml
``` 
(and similarly for TLS_KEY)

that way you don't need any specific task or knowledge to deploy new certificate.

Reply:
🤔  I was just thinking about using this command on each deploy. It will do the same.

Reply:
changed in https://github.com/shopsys/demoshop/pull/84/commits/e092ab491ffd8191ef8c154b5e8cbbe0bd0cb72a now it seems much better 👍